reach4help/reach4help
# Update impactPage

## Type of Changes

|      | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: New feature     |
|  ✓    | :link: Update dependencies |

## Description
**This pull request**:
1- Adds new dependencies: `react-datamaps` and `d3`.
2- Moves the `worldMap` component from `web-client` to `site`.
3- Modifies `path/to/impactPage.js` to display the map.
4- Adds new supporters to the partners section and removes the old ones.

![map](https://i.ibb.co/MkwwkPt/map.jpg)

## Testing Guidelines
* From your terminal, pull down this branch with `git pull origin site-design-update-impact` and check that branch out with `git checkout site-design-update-impact` 
* Then `yarn` to install the newly added dependencies locally and `yarn start` to launch the app.
* Go to [http://localhost:8000/](http://localhost:8000/) to see the map on the landing page.

**Please let me know if anything needs to change. Thanks.**
